util/helpfultriggerwrappers.py

```diff
@@ -0,0 +1,42 @@
+from enum import Enum, auto
+
+from typing import Callable
+from commands2.button import Button
+from wpilib import Joystick
+
+
+Axis = Callable[[], float]
+
+
+class AxisButton(Button):
+    """a trigger that can be fired by an axis hitting a certain limit"""
+
+    def __init__(self, axis: Axis, threshold: float) -> None:
+        self.axis = axis
+        self.threshold = threshold
```

Comment:
I would think there is a standard threshold that is good for the trigger being a button, idk why it would vary

Reply:
Variable trigger depth, multiple actions depending on how much the trigger is pressed

Reply:
For mentors or whoever is seeing this, we discussed on discord and basically decided that we're gonna use 1 setting but it leaves us the option of using others if we need to (but it's not ideal)

Reply:
*its just like deadband*